### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by AnyoneClown**

### CODE REVIEW

**Comment by AnyoneClown** [comment]
image

File: app/main.py [hunk 1]
```diff
@@ -1,44 +1,63 @@
...
+    )
+
+    await run_parallel(
+        service.send_msg(Message(speaker_id, MessageType.SWITCH_OFF)),
+        service.send_msg(Message(hue_light_id, MessageType.SWITCH_OFF)),
+        run_sequence(
+            service.send_msg(Message(toilet_id, MessageType.FLUSH)),
+            service.send_msg(Message(toilet_id, MessageType.CLEAN)),
+        ),
+    )
```
Comment (viktoria-rybenchuk) [inline app/main.py:55]: Where do you unregister device?

**Review by viktoria-rybenchuk — CHANGES_REQUESTED** [review]

**Comment by AnyoneClown** [comment]
image

**Review by Polyakiv-Andrey — APPROVED** [review]
Good job!

**Pull request closed without merging**